docs/tips/comparison.md

```diff
@@ -0,0 +1,32 @@
+# Envd vs. Other tools
+
+## What is envd?
+
+envd is a tool to build portable environment for machine learning and data science scenarios, with Python-like syntax. 
+
+
+## envd Benefits
+Benefits of using envd include:
+- Clear dev environment setup and management for each project. No longer need to worry about the conflict of CUDA, system package, virtualenv, and so on. Each project will have its own dev environment. And you can easily use the environment on any machines, and share it with your collaborators. 
+- Unify dev and production environments by envd's multi-target supports.
+- Integrate with container ecosystem. envd can be used in any container, such as Docker, Kubernetes, and so on.
+- Dataset support. envd can be used to download and preprocess datasets.
+
+## How does it work?
+- Parallize the image build process. Powered by buildkit, envd can parallize different tools installation to minimize the build time.
+- Cache support for common installation tools, such as pip, conda, apt, and so on. envd had builtin cache for these tools, which minimize the time to wait when you make modifications to your environment.
```

Comment:
I think we can emphasize how bulidkit cache is better than the normal docker build cache.

Reply:
Added a new sentence to describe normal docker build experience